addons/web/static/src/search/with_search/with_search.js

```diff
@@ -73,7 +73,7 @@ WithSearch.props = {
     context: { type: Object, optional: true },
```

Comment:
Use singular when needed: for example
 
expects the props -->  expects the prop

Reply:
Commit message updated :wink: